lib/administrate/field/date_time.rb

```diff
@@ -3,6 +3,21 @@
 module Administrate
   module Field
     class DateTime < Base
+      def to_s
+        if data.nil?
+          ""
+        else
+          data.strftime(format_date)
+        end
+      end
+
+      def format_date
+        options[:format] || "%FT%T%:z"
```

Comment:
Is there an out of the box format style we could use that Rails already defines?